Project.toml

```
@@ -43,4 +43,4 @@ StaticArrays = "0.12, 1"
 TaylorIntegration = "0.9 - 0.10"
 TaylorModels = "0.6"
 TaylorSeries = "0.12"
-julia = "1.5"
+julia = "1.6"
```

```suggestion
julia = "1.9"
```
or just `julia = "1"` -- i dont see a good reason to keep track of this lower bound. end users should be using the latest julia version.

---

I agree that this version is less important than the package versions, but it still gives the information that currently the code is compatible up to v1.6. Most packages do that I think. And v1.6 should be pretty stable anyway. Newer Julia versions rarely introduce new functions these days.

I'll merge because it is an improvement, and you can change it to `1` if you prefer that.